logger.go

```diff
@@ -52,6 +52,8 @@ func init() {
 // The first call to Init populates the default logger and returns the
 // generated logger, subsequent calls to Init will only return the generated
 // logger.
+// If the logFile passed in also satisfy io.Closer, logFile.Close will be called
```

Comment:
'If the logFile passed in is an io.Closer,...'

Reply:
Done,.